docs/source/project_documentation/rest-api.md

```diff
@@ -0,0 +1,213 @@
+# Rest-API Plugin
+
+The Rest-API plugin facilitates the use of the AaC application through a `RESTful API` interface. A `RESTful API` is an architectural style where an application interface sends and receives data through HTTP requests.
+
+## Help command for Rest-API
```

Comment:
```suggestion
## `rest-api` Command
### Help Command for `rest-api`
```

So that we stay consistent on sectioning, it's plugin, command, then command info. Repeat for each command within the plugin. 

You will have to adjust headings level remainder of content. (i.e. the '#' preceding heading title)